### PR TITLE
[docu] Mention option to set build dir via Rootrc

### PR DESCRIPTION
**Pull request by ferdymercury**
# This Pull request:

## Changes or fixes:
Mention in docu option to set build dir via rootrc
https://github.com/root-project/root/issues/10147

## Checklist:

- [ ] tested changes locally
- [x] updated the docs (if necessary)

### CODE REVIEW

File: core/base/src/TSystem.cxx [hunk 1]
```diff
@@ -4012,16 +4012,20 @@ const char *TSystem::GetObjExt() const
...
+/// in sub-directories of 'build_dir' including the full pathname
+/// of the script. If the script is located at `/full/path/name/macro.C`
+/// the library will be located at `build_dir+/full/path/name/macro_C.so`
 /// If 'isflat' is true, then no subdirectory is created and the library
-/// is created directly in the directory 'build_dir'.  Note that in this
+/// is created directly in the directory 'build_dir'. Note that in this
 /// mode there is a risk than 2 script of the same in different source
 /// directory will over-write each other.
+/// \note This `build_dir` can also be controlled via `ACLiC.BuildDir` in
+/// your `.rootrc`, see
```
Comment (pcanal) [inline core/base/src/TSystem.cxx:4028]: Is something missing after the 'see'?

Comment (ferdymercury) [reply core/base/src/TSystem.cxx]: oops. corrected. thanks.

**Review by pcanal — APPROVED** [review]

**Comment by pcanal** [comment]
thanks.

**Comment by couet** [comment]
@ferdymercury : Can we merge this one ?

**Comment by ferdymercury** [comment]
From my side, yes :)

**Comment by couet** [comment]
Ok, as @pcanal approved it, let's go on.
Thanks

**Pull request merged**